### ISSUE TITLE
[Keyboard Navigation – Make Code - Import File] Keyboard focus indicator is not visible on 'Go ahead' under 'Import File' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3568]
**User Experience**:
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to import file option and navigate through the all the controls of dialog.
6. Verify that keyboard focus indicator is visible on 'Go ahead' button or not.

**Actual Result:**
Keyboard focus indicator is not visible on 'Go ahead' button under 'Import File' pop-up window and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on 'Go ahead' button under 'Import File' pop-up window, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**Note:**
1. Same issue appear under Chrome & Firefox.
2. Same issue exists under 'Add custom blocks' pop-up window.
3. Same issue exists throughout the application where this type of control appears.

**WCAG Reference:**
[WCAG 2.4.7 - Focus Visible
](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.7, #Win10-Edge(Chromium)

[WCAG2.4.7_Import File_Go ahead.zip](https://github.com/microsoft/pxt-microbit/files/5480121/WCAG2.4.7_Import.File_Go.ahead.zip)

**Comment by jwunderl** [comment]
By default the buttons darken for selection per sui styling, in high contrast mode they display focus outline.
![image](https://user-images.githubusercontent.com/5615930/113221493-fec44980-9239-11eb-93ab-283b264a9e74.png)
